src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseHybridFVM.cpp

```diff
@@ -260,6 +260,18 @@ void SinglePhaseHybridFVM::assembleFluxTerms( real64 const dt,
 
 }
 
+
+void SinglePhaseHybridFVM::assembleStabilizedFluxTerms( real64 const dt,
+                                                        DomainPartition const & domain,
+                                                        DofManager const & dofManager,
+                                                        CRSMatrixView< real64, globalIndex const > const & localMatrix,
+                                                        arrayView1d< real64 > const & localRhs )
+{
+  // pressure stabilization not implemented
+  return assembleFluxTerms( dt, domain, dofManager, localMatrix, localRhs );
```

Comment:
i would avoid doing this and simply crash here

Reply:
I agree. Or maybe add a warning and continue? but throwing an error is probably better.

Reply:
Good point, I went in and threw errors for the unsupported functions